.gitlab-ci.yml

```diff
@@ -75,53 +75,32 @@ build_puppy_agent-deb_x64:
   script:
     - inv -e agent.build --puppy
 
-# build dogstatsd for deb-x64
```

Comment:
This is from https://github.com/DataDog/datadog-agent/pull/747 and should be kept, for cloudfoundry packages

Reply:
Oh, sorry, I guess I failed my rebase 🤦‍♂️